python/destinations/Amazon-Timestream/main.py

```diff
@@ -0,0 +1,124 @@
+import os
+
+import boto3
+from botocore.config import Config
+from botocore.exceptions import ClientError
+from quixstreaming import QuixStreamingClient, StreamReader, ParameterData, ParameterValueType
+from quixstreaming.app import App
+from quixstreaming.models.parametersbufferconfiguration import ParametersBufferConfiguration
+
+# Quix injects credentials automatically to the client. Alternatively, you can always pass an SDK token manually as an argument.
+client = QuixStreamingClient()
+
+print("Opening input topic")
+input_topic = client.open_input_topic(os.environ["input"])
+
+config = Config(
+    region_name=os.environ["region"]
+)
+
+timestream = boto3.client(
+    "timestream-write",
+    aws_access_key_id=os.environ["aws_access_key_id"],
+    aws_secret_access_key=os.environ["aws_secret_access_key"],
+    config=config
+)
+
+database_name = os.environ["database_name"]
+batch_size = int(os.environ["batch_size"])
+
+if batch_size < 1 or batch_size > 100:
+    print("Invalid batch_size set. Must be between 1 to 100")
+    exit(0)
+
+if database_name == "quix":
```

Comment:
@anandshah123 Is this string constant intentional?

Reply:
Good catch. Old code. Updated now to auto create any database name